tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -1905,6 +1905,57 @@ public function testSSRLogs(): void
 
         $this->assertNotEquals($log1Id, $log2Id);
 
+        $site = $this->updateSite(
+            [
+                '$id' => $siteId,
+                'name' => 'SSR site',
+                'framework' => 'astro',
+                'adapter' => 'ssr',
+                'buildRuntime' => 'node-22',
+                'outputDirectory' => './dist',
+                'buildCommand' => 'npm run build',
+                'installCommand' => 'npm install',
+                'fallbackFile' => '',
+                'logging' => false // set logging to false
+            ]
+        );
+        $this->assertEquals(200, $site['headers']['status-code']);
+        $response = $proxyClient->call(Client::METHOD_GET, '/logs-inline');
+        $this->assertEquals(200, $response['headers']['status-code']);
+        $this->assertStringContainsString("Inline logs printed.", $response['body']);
+
+        $logs = $this->listLogs($siteId, [
+            Query::orderDesc('$createdAt')->toString(),
+            Query::limit(1)->toString(),
+        ]);
+        $this->assertEquals(200, $logs['headers']['status-code']);
+        $this->assertStringContainsString("GET", $logs['body']['executions'][0]['requestMethod']);
+        $this->assertStringContainsString("/logs-inline", $logs['body']['executions'][0]['requestPath']);
```

Comment:
Lets assertEqual here. using stringContainsString is bad practice, should only be done when required